### ISSUE TITLE
[Raid] Trial of the Crusader Anub'arak

### ISSUE THREAD

**Issue by CrustyMuffin** [WoWManiaUK/Redemption#2004]
The issue with this boss (http://www.wowhead.com/npc=34564/anubarak) is, that he will bug after the first wipe against him. The issue doesn´t seem to appear, if we die during his first Phase, it only appears in the 2nd and 3rd Phase. If we wipe while he is in Phase 2, he will stay underground after all players are dead and we can´t fight, attack or target him anymore. If we wipe im Phase 3 while he does his "Leeching Swarm" (http://www.wowhead.com/spell=66118/leeching-swarm), he will keep this ability active after we all died and when we enter again, this ability will kill us before we can even start the bossfight. This issue appears in  both the normal and heroic version of the encounter.

**Comment by fvicaria** [comment]
There is already an issue right below. Please search before creating a new issue.

**Comment by CrustyMuffin** [comment]
that is a different bug from the same boss

**Comment by CrustyMuffin** [comment]
please read both reports

**Comment by fvicaria** [comment]
Do not open multiple reports on the same boss.
If the problem is an NPC fine if not just comment on the same.

**Comment by CrustyMuffin** [comment]
ok, i´m going to comment this into the other report

**Comment by fvicaria** [comment]
Thanks. It makes it easier for the developer.
